UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -259,7 +264,8 @@ public void BodyPartAddHealthMaster(LivingHealthMasterBase livingHealth) //Only
 
 			SetHealthMaster(livingHealth);
 			BodyType bodyType = BodyType.NonBinary;
-			if (playerSprites.ThisCharacter != null)
+			if (isCybernetic == true && playerSprites.ThisCharacter.Species != "Cyborg") bodyType = BodyType.Other2; //Synthetic
```

Comment:
this is technically some of the auto init code for players getting used ,  probably a better way is when you make a cyborg arm it sets the Gender automatically so it doesn't think it's unset

Reply:
But confused what you mean on this one.
So explanation why this is here, cyborg limbs, as im sur eyou know, determine their variant based on the body type. Its how engi and janitor cyborgs exist. Problem is, if you implanted a cyborg limb on like a male human, it would have engi borg sprites because the body type of the organic. This here checks if the limb is cybernetic, but the host organic, look it to have the organic cyborg limb sprites.

Reply:
well what I'm saying is that, you set the initial sprites  when it's made through machinery, 
so it doesn't get auto set when implanted

Reply:
Body parts always have their sprites set to match the body once implanted, its not cyborg specific. Even if you set the body part type before implanting it'll get overridden once implanted. This line exists to disable that autoset in the case of cyber => organic implants

Reply:
that shouldn't be happening, Once it set with sprites it should be keeping those

Reply:
The unchanged code is:

BodyType bodyType = BodyType.NonBinary;
if (playerSprites.ThisCharacter != null)
{
	bodyType = playerSprites.ThisCharacter.BodyType;
}

Whenever it was added to a new body, its Sprites were set to match that, no matter what.

Also even if wasnt the case, the solution you proposed would work as they NEED to change when implanted to cyborgs. So in either scenario code here is needed.

Reply:
The thing is, the cyborg NonBinary sprites aren't the ones that are needed for humans. Basically, if you print a cyborg leg, it should be able to implanted on a cyborg or a human without being strange.
You should have to take the carousel out, and put it back in each time you want to add more legs to an Engiborg for instance. They should change to match the borg they are implanted on. And if implanted on an organic, they should be the humanoid borg sprite.
There shouldnt be a specific object for humans, one for borgs, one for engi borgs etc... that goes against the whole point of unifying synth and cyborg limbs, as it was just confusing

Reply:
well the problem with the prosthetic arm versus Robotic arm 
prosthetic arm has a hand, cyborgs don't have hands

Reply:
so you need something to differentiate between the two

Reply:
Cyborgs have hands as part of this, it's more of an unintended consequence of the unification, but it never made sense that for some arbritary reason you could implant hands onto cyborgs in the first place.

Reply:
well the entire gameplay poiint of cyborgs as they don't have hands they have the tool Carousel instead, meant to differentiate between playing a normal character

Reply:
They still have a tool carousel, but arms give hands in unitystation. If the intent is that cybirgs that spawn in dont have hand slots, then they shouldn't spawn in with arms. Otherwise they arent arms they are just slabs on metal attached to their side.
Again this is because we have no distinction between hands and arms as body aprts (cant surgically remove hands without removing arms). But as long as that's the case, cyborg arms should give hand slots

Reply:
Also, cyborgs still dont have back slots, chest slots, belts, pockets etc. Giving them hands does not ruin cyborgs, and allows then to do stuff like push buttons. The tool carousel is still very valuable for them

Reply:
hummm, it's just departing quite a big way from ss13 cyborgs